pandas/tests/io/json/test_pandas.py

```diff
@@ -133,15 +133,21 @@ def test_frame_non_unique_index_raises(self, orient):
             [[Timestamp("20130101"), 3.5], [Timestamp("20130102"), 4.5]],
         ],
     )
-    def test_frame_non_unique_columns(self, orient, data):
+    def test_frame_non_unique_columns(self, orient, data, request):
+        if isinstance(data[0][0], Timestamp) and orient == "split":
+            mark = pytest.mark.xfail(
+                reason="GH#55827 non-nanosecond dt64 fails to round-trip"
```

Comment:
FYI there is active work on deprecating this in https://github.com/pandas-dev/pandas/pull/57987